### ISSUE TITLE
The 'add-path' & 'set-env' command will be disabled on november 16th

### ISSUE THREAD

**Issue by sp0ker** [google-github-actions/setup-gcloud#226]
Hello,

I am starting to use your Github Action today and I noticed that you are using a deprecated and delete soon param.
Here the output from the workflow.

```bash
/bin/tar xz --warning=no-unknown-keyword -C /home/runner/work/_temp/a8da1019-924a-476f-a91b-970ea10ad7ee -f /home/runner/work/_temp/a68ebee4-749d-48df-96f1-949f382d65c7
Error: The `add-path` command is deprecated and will be disabled on November 16th. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
Error: The `set-env` command is deprecated and will be disabled on November 16th. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/
```

Here my code to invoke your action
```yaml
- name: Setup gcloud environment
        uses: google-github-actions/setup-gcloud@0.1.3
        with:
          service_account_key: ${{ secrets.BUILDPUBLISHCLOUDRUN_KEY }}
```

My job is working, I am just curious about this output.

Thank you :)

**Comment by bharathkkb** [comment]
Thanks for the report @sp0ker 
We have not released a new version that removes the deprecated command, but it has been fixed on main - `google-github-actions/setup-gcloud@master`. We will release a new soon.

**Comment by jwtracy** [comment]
Seeing some deprecation warnings in my up-to-date fork due to what I presume to be lag in upstream changes, https://github.com/actions/toolkit/issues/630. Some actions here depend on tool-cache which is not yet updated.

**Comment by bharathkkb** [comment]
Hi @jwtracy 
Yup looks like you are right: https://github.com/actions/toolkit/pull/591

**Comment by bharathkkb** [comment]
Hi @sp0ker 
We have released [v0.2.0](https://github.com/google-github-actions/setup-gcloud/releases/tag/v0.2.0) with the fix. We also encourage you to [migrate](https://github.com/google-github-actions/setup-gcloud/tree/master/setup-gcloud#-deprecation-notice) to `google-github-actions/setup-gcloud` for new features and fixes. Feel free to reopen if needed.
